core/src/dispatch/mod.rs

```diff
@@ -139,7 +139,7 @@ impl Default for NetworkConfig {
         NetworkConfig {
             addr: "127.0.0.1:0".parse().unwrap(),
             transport: Transport::TCP,
-            buffer_config: BufferConfig::default(),
```

Comment:
Actually I think it's a good idea to use `default` here for consistency, even if `new` and `default actually do the same thing.`